sdk/core/core/inc/az_http_pipeline.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_PIPELINE_H
+#define AZ_HTTP_PIPELINE_H
+
+#include <az_contract.h>
+#include <az_http_policy.h>
+#include <az_http_request.h>
+#include <az_pair.h>
+#include <az_span.h>
+#include <az_str.h>
```

Comment:
you only need to include az_result.h, az_http_request_builder.h, az_mut_span.h

Reply:
done